src/lava/proc/rf/process.py

```diff
@@ -0,0 +1,54 @@
+"""Code based on the tutorial for now:
+https://github.com/lava-nc/lava/blob/main/tutorials/in_depth/tutorial02_processes.ipynb
+
+"""
+import numpy as np
+import typing as ty
+from lava.magma.core.process.process import AbstractProcess, LogConfig
+from lava.magma.core.process.variable import Var
+from lava.magma.core.process.ports.ports import InPort, OutPort
+
+
+class AbstractRF(AbstractProcess):
+    def __init__(self, 
+            shape: ty.Tuple[int, ...],
+            sin_decay: float,
+            cos_decay: float, 
+            vth: float,
+            name: str,
+            log_config: LogConfig) -> None:
```

Comment:
Let's add an additional argument state called `state_exp = 0`, similar to sigma-delta neurons : https://github.com/lava-nc/lava/blob/main/src/lava/proc/sdn/process.py